apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -325,14 +345,17 @@ export default function LearningGoals({
             isAutosaving={autosaveStatus === STATUS.IN_PROGRESS}
           />
           {learningGoals[currentLearningGoal].tips && !isStudent && (
-            <div>
-              <Heading6>{i18n.tipsForEvaluation()}</Heading6>
-              <div className={style.learningGoalTips}>
+            <details>
+              <summary>
+                <strong>{i18n.tipsForEvaluation()}</strong>
```

Comment:
Just for my own learning here, why did you go with `strong` vs. `Heading6`?

Reply:
The heading styles create a new line, which messed with the automatic styling of the summary element (made the dropdown triangle appear above the text instead of next to it. Changing the styling of summary removes the triangle, so it would require more code to make this work with Heading6 than without. (Also, this may be moot since the tips might be removed in the future!)